src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -61,6 +61,8 @@ struct Input
   int circlePoints {100};
   RuntimePolicy policy {RuntimePolicy::seq};
 
+  double distThreshold {1e100};
```

Comment:
Any reason not to make this default to `std::numeric_limits<double>::max()` ?

Reply:
The reason is that we'd have to square it and compare the squares -> floating point overflow.  We could set it to sqrt(numeric_limits<double>::max()) - epsilon.  But I thought that would be a weird number that may be confusing to look at in a debugger.  What do you think?

Reply:
It would be good to check that passing in `numeric_limits<double>::max` doesn't do anything stange.

Does the following help test against expected behavior?

```cpp
#include <iostream>
#include <limits>


int main () {

  double max_1 = std::numeric_limits<double>::max();
  double max_2 = max_1 * max_1;

  std::cout << "numeric limits max: " << max_1 << '\n';
  std::cout << "numeric_limits max squared: " << max_2 << '\n';

  for(double d : {-5., 0., 100., std::numeric_limits<double>::max(), max_1, max_2})
  {
    for(double m : {max_1, max_2})
    {
        if( d < m) { std::cout << d << " < " << m << std::endl; }
        else if( d == m) { std::cout << d << " == " << m << std::endl; }
        else if( d > m) { std::cout << d << " > " << m << std::endl; }
    }
  }

  return 0;
}
```
https://godbolt.org/z/99rzoTao7

Here is the output of some comparisons (using gcc@12.1):
```
numeric limits max: 1.79769e+308
numeric_limits max squared: inf
-5 < 1.79769e+308
-5 < inf
0 < 1.79769e+308
0 < inf
100 < 1.79769e+308
100 < inf
1.79769e+308 == 1.79769e+308
1.79769e+308 < inf
1.79769e+308 == 1.79769e+308
1.79769e+308 < inf
inf > 1.79769e+308
inf == inf
```

That might imply that if we pass in `numeric_limits<double>::max()` then our comparisons for squared distances will be against `inf`.

Reply:
Yes.  That looks like using `max()` should work.  Thanks!